src/core/jwt/jwtTokenStore.js

```diff
@@ -56,6 +64,9 @@ const jwtTokenStoreFactory = function jwtTokenStoreFactory({namespace = 'global'
          * @returns {Promise<string|null>} stored access token
          */
         getAccessToken() {
+            if (accessTokenTTL && accessTokenStoredAt + accessTokenTTL < Date.now()) {
+                return Promise.resolve(null);
```

Comment:
suggestion: shouldn't you clear it from the store if the TTL is expired ? 
```suggestion
                return this.clear().then( () => null);
```

Reply:
I can clear it, but it makes no sense. This condition will always be true, so it will return with `null`. If I would clear it `accessTokenStoredAt + accessTokenTTL < Date.now()` is true (even if I set `accessTokenStoredAt` to 0), so it will repeat the clear every time when somebody wants to get the token. BTW, the handler automatically refreshes the token if `getToken` does not return with a token, so the store would be empty for milliseconds.

Reply:
ok then